metaflow/plugins/aip/aip_exit_handler.py

```diff
@@ -69,11 +69,13 @@ def email_notify(send_to):
         argo_url_prefix = get_env("ARGO_RUN_URL_PREFIX", "")
         k8s_namespace = get_env("POD_NAMESPACE", "")
 
-        argo_ui_url = f"{argo_url_prefix}/argo-ui/workflows/{k8s_namespace}/{run_id}"
+        argo_ui_url = (
+            f"{argo_url_prefix}/argo-ui/workflows/{k8s_namespace}/{argo_workflow_name}"
```

Comment:
Ah this is because the `run_id == argo_workflow_name` now it's guaranteed to be unique?

Reply:
They are not equal, run_id is now unique with the uid while argo_workflow_name is not. 

Argo UI 3.4.x doesn’t support a uid in the url while it’s still running. Created https://zillowgroup.atlassian.net/browse/AIP-8095 to try this in 3.5.x and if it doesn’t support uid in the url then create an Argo bug.